opslevel/resource_opslevel_team.go

```diff
@@ -108,17 +118,17 @@ func (teamResource *TeamResource) Schema(ctx context.Context, req resource.Schem
 				Description: "A description of what the team is responsible for.",
 				Optional:    true,
 			},
-		},
-		Blocks: map[string]schema.Block{
-			"member": schema.ListNestedBlock{
-				NestedObject: schema.NestedBlockObject{
+			"member": schema.SetNestedAttribute{
+				Description: "Unordered list of members. Only manages team members that were defined in terraform.",
+				Required:    true,
+				NestedObject: schema.NestedAttributeObject{
 					Attributes: map[string]schema.Attribute{
 						"email": schema.StringAttribute{
 							Description: "The email address of the team member. Must be sorted by email address.",
 							Required:    true,
 						},
 						"role": schema.StringAttribute{
-							Description: "The role of the team member.",
+							Description: "The role of the team member. Options: `contributor` or `manager`",
```

Comment:
Is this true?  I thought it was free form so that end users could come up with whatever roles they want?

Reply:
Will reverse this!